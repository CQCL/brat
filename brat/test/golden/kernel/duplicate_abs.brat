f :: {a :: Qubit -o a :: Qubit}
f = { a -> a }

h2 :: {x :: Qubit, y :: Qubit -o z :: Qubit}
h2 = { p, p -> f(p) }
