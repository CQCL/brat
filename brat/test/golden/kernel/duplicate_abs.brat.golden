Error in test/golden/kernel/duplicate_abs.brat@5:8
h2 = { p, p => f(p) }
       ^^^^

  Type error: Variable(s) defined twice: p

